### ISSUE TITLE
looping doesn't appear to work.

### ISSUE THREAD

**Issue by JWyllieDev** [inkle/inky#128]
Hey there, I am just getting into ink/inky, and working through the [tutorial](https://github.com/inkle/ink/blob/master/Documentation/WritingWithInk.md).
I've tried to create a **very basic** loop, and it gives a "loose ends" error. Can anyone help?

```
-> begin
=== begin ===
Hello, world!
...
You receive no response; the world does not know (or care) who you are. -> sad
===sad===
* oh... :( -> goodbyeWorld
===goodbyeWorld===
* [try again]
Let's try that again... -> begin

->end
```
Works the first time, and sends me back to _begin_, but _begin_ then throws a "loose ends" error. help?

**Comment by joethephish** [comment]
(note when I tried this, it told me the `-> end` divert target couldn't be found - I think you meant `-> END`?)

In general, the reason it's running out of content is that choices are by default "once only" - after being taken they can't be taken again. So when you loop back round, there's no content left. If you change the `*` in the choice to a `+` then it's allowed to be taken multiple times. Fixed story:

```
-> begin

=== begin ===
Hello, world!
...
You receive no response; the world does not know (or care) who you are. -> sad

===sad===
+ oh... :( -> goodbyeWorld

===goodbyeWorld===
+ [try again]
Let's try that again... -> begin

-> END
```
